src/main/java/net/sf/jabref/gui/importer/fetcher/IdBasedEntryFetcher.java

```diff
@@ -34,8 +35,8 @@ public boolean processQuery(String query, ImportInspector inspector, OutputPrint
             match.ifPresent(inspector::addEntry);
             return match.isPresent();
         } catch (FetcherException e) {
-            status.setStatus(e.getLocalizedMessage());
-            LOGGER.error("Error while fetching from" + fetcher.getName(), e);
+            LOGGER.error("Error while fetching from " + getTitle(), e);
+            ((ImportInspectionDialog)inspector).showErrorMessage(this.getTitle());
```

Comment:
You completely ignore the exception details when you call `showErrorMessage`, thus the user gets just a standard message although the FetcherException contains all the necessary details about what went wrong (`e.getLocalizedMessage`)